xbmc/guilib/GUIWindow.cpp

```diff
@@ -52,7 +52,7 @@ bool CGUIWindow::icompare::operator()(const std::string &s1, const std::string &
 
 CGUIWindow::CGUIWindow(int id, const std::string &xmlFile)
 {
-  SetID(id);
+  CGUIWindow::SetID(id);
```

Comment:
Why is the class prefix needed? The call is already happening inside `CGUIWindow`.